src/integrationTest/java/uk/gov/hmcts/reform/judicialapi/elinks/PeopleIntegrationTest.java

```diff
@@ -87,47 +84,6 @@ void verifyPeopleJrdUserProfile() {
 
     }
 
```

Comment:
Removed below test case since we have removed base_location_id default values in flyway script. Since there should no default value for base_location_type table, It will populate via elink